nextflow_schema.json

```diff
@@ -499,6 +501,28 @@
                     "default": 100,
                     "description": "Option to specify the length of the donor/acceptor sequence on each side of the junctions used in constructing the splice junctions database. By default the option is set to 100. However, we recommend setting a value depending on the read length: read/mate length - 1.",
                     "fa_icon": "fas fa-sliders-h"
+                },
+                "outWigType": {
+                    "type": "string",
+                    "default": "None",
+                    "description": "Used to generate signal outputs, such as \"wiggle\" and \"bedGraph\".",
+                    "fa_icon": "fas fa-sliders-h"
+                },
+                "outWigStrand": {
+                    "type": "string",
+                    "default": "Stranded",
+                    "description": "Options are Stranded or Unstranded when defining the strandedness of wiggle/bedGraph output",
```

Comment:
Please use the `enum` option in the schema to limit the options that can be entered if there are only specific valid values.

This can be done via the little settings cog on the right hand side in the `nf-core schema build` GUI.